lib/instrumentation/dropped-span-stats.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+class DroppedSpanStats {
+  constructor () {
+    this.statsMap = new Map()
+  }
+
+  captureDroppedSpan (span) {
+    const resource = span && span._destination && span._destination.service && span._destination.service.resource
+    if (!resource || !span.outcome || this.statsMap.size === 128) {
+      return
+    }
+
+    const stats = this.getOrCreateStats(resource, span.outcome)
+
+    stats.duration.count++
+    stats.duration.sum.us += (span._duration * 1000)
+    return true
```

Comment:
Why return a true/false for this `captureDroppedSpan()`? (FWIW the Java impl doesn't.)

Reply:
Sends a signal as to whether the operation succeeded in order to enable test driven development.